lib/widget/movie_poster.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:flutter/material.dart';
+import '../models/movie.dart';
+import '../utils/general_constants.dart';
+
+class MoviePoster extends StatelessWidget {
+  final Movie movie;
+  static const double heightSizeBox = 50;
+  static const double widthSizeBox = 200;
+
+  const MoviePoster({
+    super.key,
+    required this.movie,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      margin: const EdgeInsets.all(EdgeInsetsConst.edgeTen),
+      child: Column(
+        children: [
+          Expanded(
+            child: Image.network(
+              ApiEndPoints.urlImages + movie.poster,
```

Comment:
Let's create a getter method for the URL inside the `Movie` class for this!

Reply:
done